copter/source/docs/thrust_loss_yaw_imbalance.rst

```diff
@@ -0,0 +1,47 @@
+.. _thrust_loss_yaw_imbalance:
+
+======================================
+Thrust loss and yaw imbalance warnings
+======================================
+
+If you see thrust loss or yaw imbalance warnings this page outlines some checks and modifications that should be done to resolve the issue. In the majority of cases these warnings are the result of incorrect hardware selection or setup.
+
+These warnings are designed to detect hardware failures in the propulsion system, if they begin to occur on a vehicle that has been flying without warnings the propulsion system should be checked.
+Warning are more likely to happen with larger payloads and in strong/gusty winds.
+
+Potential thrust Loss
+=====================
+
+If potential thrust loss messages are seen on the GCS or in the data-flash logs investigation should be done to find the cause and remedy. The warning will give a motor number, for example:
+
+```Potential Thrust Loss (3)```
+
+These warnings are the result of a motor or motors saturating at 100% throttle. Because of this saturation ArduCopter can no longer achieve the requested roll, pitch, yaw and throttle output.
+If this continues for a prolonged period of time the vehicle will have reduced reduced altitude and attitude control and may crash.
+
+If these messages are seen in hover or relaxed flight the problem must be fixed in hardware. The vehicles thrust to weight ratio should be increased, either by a change in propulsion or reduction in mass.
+
+If only seen in climbing and aggressive manuevers it may be sufficiency to lower the requested accelerations and speeds. Again the vehicle thrust to weight ratio can be increased to allow higher accelerations and speeds.
+
+Yaw imbalance
+=============
+
+The yaw imbalance warning is a measure of how hard the vehicle is working to yaw, the warning will trigger before the yaw output is saturated. If the yaw output becomes saturated the vehicles
+ability to maintain yaw will be compromised. And, in the worst case, this will result in the vehicle rapidly spinning. The warning message gives a percentage of maximum yaw outpu. At 100% it is saturated. For example:
```

Comment:
```suggestion
ability to maintain yaw will be compromised. And, in the worst case, this will result in the vehicle rapidly spinning. The warning message gives a percentage of maximum yaw output. At 100% it is saturated. For example:
```